packages/web-react/src/components/FileUploader/useFileUploaderStyleProps.ts

```diff
@@ -68,6 +78,15 @@ export const useFileUploaderStyleProps = (props?: FileUploaderStyleProps): FileU
   const fileUploaderAttachmentImageClass = `${fileUploaderAttachmentClass}__image`;
   const fileUploaderAttachmentSlotClass = `${fileUploaderAttachmentClass}__slot`;
 
+  const { x, y, width, height } = props?.meta || {};
```

Comment:
Just in case: There is no check if these values exist. How about passing `meta = { x: 1, y:2, a:3, b:4 }`? What will happen? I think there should exist all keys or the object and their values should not be used.

Reply:
Oh, the check is a line down.

Reply:
Beware: the check should come first and the logic should apply only when the check passes. Now you are creating the `imageCropCSS` object every time.

Reply:
good catch! fixed